tests/test_capture_minidump.py

```diff
@@ -0,0 +1,52 @@
+import os
+import shutil
+
+import pytest
+
+from . import make_dsn, run, Envelope
+from .assertions import (
+    assert_attachment,
+    assert_breadcrumb,
+    assert_minidump,
+)
+from .conditions import has_http
+
+pytestmark = pytest.mark.skipif(not has_http, reason="tests need http")
+
+# fmt: off
+auth_header = (
+    "Sentry sentry_key=uiaeosnrtdy, sentry_version=7, sentry_client=sentry.native/0.7.11"
+)
+# fmt: on
+
+
+def test_capture_minidump(cmake, httpserver):
```

Comment:
Is this function sufficiently different from all other HTTP integration tests to end up in a separate module?